xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -2453,7 +2453,7 @@ CSampleBuffer* CActiveAE::SyncStream(CActiveAEStream *stream)
         CLog::Log(LOGDEBUG,"ActiveAE::SyncStream - average error %f, last average error: %f", error, stream->m_lastSyncError);
         stream->m_lastSyncError = error;
       }
-      else
+      else if (stream->m_lastSyncError / error < .0)
```

Comment:
one error interval at most. so perhaps yes, ages. on the other hand it allows (together with the convergence) to keep stable resampling even if refClock is wrong.
but beside robustness test it is stupid scenario anyhow... removed.